drivers/session_mock.go

```diff
@@ -2,17 +2,33 @@ package drivers
 
 import (
 	"context"
+	"fmt"
+	"time"
 
 	"github.com/livepeer/go-livepeer/net"
 	"github.com/stretchr/testify/mock"
 )
 
 type MockOSSession struct {
 	mock.Mock
+	waitForCh bool
+	waitCh    chan interface{}
+	back      chan interface{}
 }
 
-func (s *MockOSSession) SaveData(name string, data []byte, meta map[string]string) (string, error) {
-	args := s.Called()
+func (s *MockOSSession) init() {
```

Comment:
```suggestion
func (s *MockOSSession) NewMockOSSession() *MockOSSession {
```

We can just return a properly initialized MockOSSession.

Reply:
Fixed